AnimalPicker.java

```diff
@@ -17,11 +17,16 @@ public static void main(String[] args) {
                                         "(\\,--------'()'--o\r\n" + //
                                         " (_    ___    /~\"\r\n" + //
                                         "  (_)_)  (_)_)");
+            } else if (word.equalsIgnoreCase("fish")) {
+                System.out.println("          /\"*._         _\r\n" + //
+                                        "      .-*'`    `*-.._.-'/\r\n" + //
+                                        "    < * ))     ,       ( \r\n" + //
+                                        "      `*-._`._(__.--*\"`.\\");
             } else {
                 // error message
                 System.out.print("Invalid animal. ");
             }
-        } while (!word.equalsIgnoreCase("cat") && !word.equalsIgnoreCase("dog"));
```

Comment:
I like the way you used the do while loop, it's a smart way to do it. It means there is less code redundancy.